Cargo.toml

```diff
@@ -19,4 +24,43 @@ lexopt    = "0.3.0"
 test_bin = "0.4.0"
 
 [features]
-arbitrary = [ "dep:arbitrary", "domain/arbitrary" ]
\ No newline at end of file
+arbitrary = [ "dep:arbitrary", "domain/arbitrary" ]
+
+[package.metadata.deb]
+license-file = ["LICENSE", "0"]
+depends = "$auto"
+section = "net"
+priority = "optional"
+assets = [
+    ["target/release/dnst", "usr/bin/", "755"],
+    ["README.md", "usr/share/doc/dnst/", "644"],
+    # TODO: Extend Ploutos to generate the man pages from sources.
+    #["doc/dnst.1", "usr/share/man/man1/dnst.1", "644"],
+]
+changelog = "target/debian/changelog" # this will be generated by the pkg workflow
+copyright = "Copyright (c) 2024, NLnet Labs. All rights reserved."
+# preserve_symlinks didn't prevent error "Asset file path does not match any files"
+# Perhaps a newer version of cargo-deb than the one used by Ploutos would fix this.
+# Instead create and remove them using maintainer scripts.
+#preserve-symlinks = true
+maintainer-scripts = "pkg/debian"
+conflicts = "ldnsutils"
+
+[package.metadata.generate-rpm]
+# "BSD" alone is the 3-clause license. Inheriting "license" from above causes rpmlint to
+# complain with "invalid-license".
+# See: https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing
+license = "BSD"
```

Comment:
Here, line 53 of Cargo.toml, contains BSD. That is not SPDX license. recent rpmlint should not complain about BSD-3-Clause anymore. It should complain about BSD now. Just remove extra license for rpm?

Reply:
https://github.com/NLnetLabs/routinator/pull/979